docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,154 @@
+> **⚡ Note**: While the `stakerd` daemon is already running, it needs access to 
+> the Bitcoin wallet containing your staking transaction's private key. This 
+> setup ensures the daemon can interact with the correct Bitcoin wallet.
+
+To get started, you will need to create a configuration file. You can do this 
+by running the `dump-config` command. This will create a `config.json` file 
+in your current working directory that you'll customize with your specific 
+values:
+
+```bash
+stakercli admin dump-config
+```
+
+The configuration file below contains default settings and placeholders 
+for your Bitcoin and Babylon values. Please replace the placeholders with 
+your specific information:
+
+```toml
+[walletconfig]
+WalletName = "btc-staker"  # Replace with your Bitcoin wallet name
+WalletPass = "your-wallet-password"  # Replace with your wallet password
+
+[walletrpcconfig]
+Host = "127.0.0.1:19001"  # Bitcoin Wallet RPC endpoint
+User = "your-rpc-user"    # Replace with your Bitcoin RPC username
+Pass = "your-rpc-pass"    # Replace with your Bitcoin RPC password
+
+[chain]
+Network = "signet"  # Bitcoin network (signet for testnet)
+```
+
+Finally, start the `btcstaker` daemon, which will manage your connections to both 
+Babylon and Bitcoin nodes, monitor transactions, and handle the staking process.
+Add the location for the above created `config.json` file to the command:
+
+```shell
+stakerd --configfile <config-file-path> admin start
+```
+
+## Register your stake
+
+After configuring and starting the `btcstaker` daemon, you'll need the 
+`global_parameters.json` file (created during prerequisites) for the next steps.
+
+A few notes about the `global-parameters.json` file:
+1. It is used to retrieve covenant keys and quorum from the parameters version 
+   you staked in.
+2. It can also be used to parse your Bitcoin staking transaction to extract 
+   important information from the `OP_RETURN` data, which contains:
+   - The staking period you chose
+   - The finality provider you chose
+
+> **⚡ Note**: While these parameters can be provided manually, the global 
+> parameters file provides an easier user experience. This guide is only for 
+> valid, confirmed Bitcoin staking transactions with valid `OP_RETURN` data. If 
+> your transaction has malformed `OP_RETURN` data or isn't a confirmed staking 
+> transaction, to check your eligibility please refer to the 
+> [registration eligibility guide](registration-eligibility.md).
+
+Now that you have the `global-parameters.json` file, you can register your 
+Bitcoin stake by running the following command:
+
+```shell
+stakercli daemon stake-from-phase1 <global-parameters-file> \
+    --staking-transaction-hash <your-phase1-tx-hash> \
+    --staker-address <your-btc-address> \
+    --tx-inclusion-height <block-height>
+```
+
+Parameters:
+- `global-parameters-file`: The path to the global parameters file.
+- `your-phase1-tx-hash`: The original hash of your Bitcoin staking transaction.
```

Comment:
```suggestion
- `your-phase1-tx-hash`: The original hash of your Bitcoin staking transaction.
   This will be used to retrieve the staking transaction from your wallet.
```